package.json

```diff
@@ -5,26 +5,30 @@
   "main": "dist/index.js",
```

Comment:
to the previous point about the dist folder. if we bundle the action we do need the dist folder to be in the repo